doc/classes/AnimationNodeAnimation.xml

```diff
@@ -36,7 +36,7 @@
 			If [code]false[/code], the original animation length is respected. If you set the loop to [member loop_mode], the animation will loop in [member timeline_length].
 		</member>
 		<member name="timeline_length" type="float" setter="set_timeline_length" getter="get_timeline_length">
-			If [member use_custom_timeline] is [code]true[/code], offset the start position of the animation.
+			If [member stretch_time_scale] is [code]true[/code], scales the animation to this length.
```

Comment:
```suggestion
			The length of the custom timeline.
			If [member stretch_time_scale] is [code]true[/code], scales the animation to this length.
```

The original explanation was indeed incorrect. However, since your changes only note to stretched cases, a general explanation is required.